### PR TITLE
Serve TOCs

### PR DESCRIPTION
**Pull request by jacobwegner**
- Implements a basic file-based API to serve TOCS (https://mini-stack-a-feature-se-j47yu0.herokuapp.com/tocs/)
- Updates version metadata to include a `default_toc`

Eventually,  [As an ATLAS consumer, I can fetch relevant TOCs for a URN](https://trello.com/c/ElAdk9LU) will supersede this API to support richer querying.

### CODE REVIEW

**Comment by jacobwegner** [comment]
[As a curator, I can construct TOCs according to a specified data format](https://trello.com/c/VNwGmh1J/46-as-a-curator-i-can-construct-tocs-according-to-a-specified-data-format)

**Comment by jacobwegner** [comment]
[As a curator, I want to define the default TOC for a given URN's version](https://trello.com/c/xVl0uXmG/56-as-a-curator-i-want-to-define-the-default-toc-for-a-given-urns-version)

File: data/tocs/toc.oaf-1.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    {
+      "title": "Martyrdom of Polycarp",
+      "uri": "urn:cts:greekLit:tlg1484.tlg001.oaf-1:0-1"
+    },
+    {
+      "title": "Diognetus",
+      "uri": "urn:cts:greekLit:tlg0646.tlg004.oaf-1:1"
+    }
+  ]
+}
```
Comment (jhrr) [inline data/tocs/toc.oaf-1.json]: Cheeky whitespace.

**Review by jhrr — APPROVED** [review]
LGTM

File: data/tocs/toc.oaf-1.json (hunk 1 above)
Comment (jacobwegner) [reply data/tocs/toc.oaf-1.json]: @jhrr: was it just the missing newline (fixed), or was there something else? image

**Pull request merged**